quconot/implementations/mct_recursion.py

```diff
@@ -84,4 +84,6 @@ def num_auxiliary_qubits(self):
 
 
 if __name__ == "__main__":

```

Comment:
remove in final pull request

Reply:
noted, i will remove all the main

Reply:
I have pushed the changes with the last commit